### ISSUE TITLE
Terminal can't load or take long time to load

### ISSUE THREAD

**Issue by gossotjeanbaptiste** [dotnet/vscode-dotnet-runtime#1705]
Type: <b>Bug</b>

I've just install .NET Install Tool one day and since that day my terminal take very long time to load. I think it's also my computer that's slow af, you'll see the configuration and the OS that I have.

Extension version: 2.0.2
VS Code version: Code 1.87.1 (1e790d77f81672c49be070e04474901747115651, 2024-03-06T00:21:36.258Z)
OS version: Windows_NT x64 10.0.26063
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Athlon Silver 3050U with Radeon Graphics    (2 x 2296)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: disabled_off<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|13.92GB (8.29GB free)|
|Process Argv|--crash-reporter-id 97e6c0b7-bd3b-47ca-9e18-f0439e535934|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscoreces:30445986
vscod805:30301674
binariesv615:30325510
vsaa593:30376534
py29gd2263:30899288
vscaac:30438847
c4g48928:30535728
azure-dev_surveyone:30548225
2i9eh265:30646982
962ge761:30959799
pythongtdpath:30769146
pythonidxpt:30866567
pythonnoceb:30805159
asynctok:30898717
pythontestfixt:30902429
pythonregdiag2:30936856
pyreplss1:30897532
pythonmypyd1:30879173
pythoncet0:30885854
pythontbext0:30879054
dsvsc016:30899300
dsvsc017:30899301
dsvsc018:30899302
pydisww1:30981707
cppperfcontrol:30979541
d34g3935:30971562
fegfb526:30981948
bg6jg535:30979843
dsvsc020:30976470

```

</details>

**Comment by nagilson** [comment]
I'm not sure what we can do about this, installing .net is expensive and most computers should handle it ok. But after the first time it finishes downloading it shouldnt really impact things anymore until theres an update?

**Comment by gossotjeanbaptiste** [comment]
I don't really know if this is really a bug or not because I used the Bisect extension that deactivated some extension to know which extension is producing the problem and it tell me it was the .NET Installer Tool but I think this just a bug with my VS Code installation and extension mixed with the small configuration I have on the computer I had the bug because on my other computer it wasn't taking that long to load with the same extension installed.
For example my terminal could take between 5 to 15 minutes to load.

**Comment by gossotjeanbaptiste** [comment]
Yes he told me he had finished downloading but I don't remember having a path given to me.
And yes it does that everytime I loaded a new terminal like sometimes I had to restart my computer or VS Code to make everything work again because it was making my computer lagging af. But like I said it's maybe the computer who's a bit slow because on my other one with the same extension it was taking like 2 to 5 seconds to load a terminal. Sometimes if I remember correctly I'd get a report of the .NET extension telling me it was downloading something but I never really cared enough to really look at why the report where saying.
